docs/docs/cmd/docs.md

```diff
@@ -0,0 +1,31 @@
+# docs
+
+Opens CLI for Microsoft 365 docs webpage in the default browser.
```

Comment:
Usually our command descriptions don't end on a period

Reply:
I have noticed that the statements in app-open.md end with a period, but I have removed it from docs.md. Should I also remove it from app-open.md and any other commands as well?